python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -120,3 +121,49 @@ async def create_workflow_status(
         color=form.color,
         workflow=workflow,
     )
+
+
+################################################
+# update workflow status
+################################################
+
+
+@routes.stories.patch(
+    "/projects/{project_id}/workflows/{workflow_slug}/statuses/{status_slug}",
```

Comment:
the main resource in the URL doesn't need to be marked.
Examples:
```python
 @routes.projects.delete(
     "/projects/{id}",

@routes.stories.get(
    "/projects/{project_id}/stories/{ref}",

```

according to these examples, the URL could be:
> "/projects/{project_id}/workflows/{workflow_slug}/statuses/{slug}",

However, this is not a strong convention, so it's not mandatory to change it to approve the PR.

Reply:
I'm ok with that. I'll use simply "slug" for the main context parameter.